src/matrixroomhandler.ts

```diff
@@ -74,19 +96,157 @@ export class MatrixRoomHandler {
+              msgtype: "m.notice",
+              body: "How to bridge a Discord guild:\n" +
+              "1. Invite the bot to your Discord guild using this link: " + link + "\n" +
+              "2. Invite me to the matrix room you'd like to bridge\n" +
+              "3. Open the Discord channel you'd like to bridge in a web browser\n" +
+              "4. In the matrix room, send the message `!discord bridge <guild id> <channel id>` " +
+              "(without the backticks)\n" +
+              "   Note: The Guild ID and Channel ID can be retrieved from the URL in your web browser.\n" +
+              "   The URL is formatted as https://discordapp.com/channels/GUILD_ID/CHANNEL_ID\n" +
+              "5. Enjoy your new bridge!",
+          });
+      } else if (command === "bridge") {
+          if (context.rooms.remote) {
+              return this.bridge.getIntent().sendMessage(event.room_id, {
+                  msgtype: "m.notice",
+                  body: "This room is already bridged to a Discord guild.",
+              });
+          }
+
+          const minArgs = 2;
+          if (args.length < minArgs) {
+              return this.bridge.getIntent().sendMessage(event.room_id, {
+                  msgtype: "m.notice",
+                  body: "Invalid syntax. For more information try !discord help bridge",
+              });
+          }
+
+          const guildId = args[0];
+          const channelId = args[1];
+          return this.discord.LookupRoom(guildId, channelId).then((result) => {
+              log.info("MatrixRoomHandler", `Bridging matrix room ${event.room_id} to ${guildId}/${channelId}`);
+              return this.bridgeMatrixRoom(result.channel, event.room_id);
+          }).then(() => {
+              return this.bridge.getIntent().sendMessage(event.room_id, {
+                  msgtype: "m.notice",
+                  body: "I have bridged this room to your channel",
+              });
+          }).catch((err) => {
+              log.error("MatrixRoomHandler", err);
+              this.bridge.getIntent().sendMessage(event.room_id, {
+                  msgtype: "m.notice",
+                  body: "There was a problem bridging that channel - has the guild owner approved the bridge?",
+              });
+          });
+      } else if (command === "unbridge") {
+          const remoteRoom = context.rooms.remote;
+
+          if (!remoteRoom) {
+              return this.bridge.getIntent().sendMessage(event.room_id, {
+                  msgtype: "m.notice",
+                  body: "This room is not bridged.",
+              });
+          }
+
+          if (!remoteRoom.data.manual_bridge) {
+              return this.bridge.getIntent().sendMessage(event.room_id, {
+                  msgtype: "m.notice",
+                  body: "This room cannot be unbridged.",
+              });
+          }
+
+          this.bridge.getRoomStore().removeEntriesByRemoteRoomId(remoteRoom.getId()).then(() => {
```

Comment:
Catch case where we fail to remove the entry?

Reply:
FWIW, I have no idea if this CAN fail but I have been bitten by not catching things before.